Makefile

```diff
@@ -5,10 +5,10 @@ PROJECT_NAME := provider-jet-template
 PROJECT_REPO := github.com/crossplane-contrib/$(PROJECT_NAME)
 
 export TERRAFORM_VERSION := 1.0.11
-export TERRAFORM_PROVIDER_SOURCE :=
-export TERRAFORM_PROVIDER_VERSION :=
-export TERRAFORM_PROVIDER_DOWNLOAD_NAME :=
-export TERRAFORM_PROVIDER_DOWNLOAD_URL_PREFIX :=
+export TERRAFORM_PROVIDER_SOURCE := hashicorp/hashicups
```

Comment:
We may, at some point, rephrase the terrajet guide: "Replace `export TERRAFORM_PROVIDER_SOURCE := hashicorp/hashicups`" in Makefile, ...